assembly/build.gradle

```diff
@@ -46,10 +46,19 @@ clean.doLast {
   cleanProduct()
 }
 
+task generateRLibs {
+  delete file("${sparkProjectRootDir}/R/lib")
+  if (!rootProject.hasProperty('R.skip')) {
```

Comment:
should we also check whether the assigned value to this property is `true`? I noticed that we are not checking that for other properties like `snappydata.enterprise`.

Reply:
Either will do. I think that was done to avoid two checks (existence and value).